superset/db_engine_specs/bigquery.py

```diff
@@ -127,7 +127,7 @@ class BigQueryEngineSpec(BaseEngineSpec):  # pylint: disable=too-many-public-met
 
     allows_hidden_cc_in_orderby = True
 
-    supports_catalog = True
+    supports_catalog = False
```

Comment:
Changing to false here and in other DB engine specs, since the specs should only have it set to true if they implement their own `get_default_catalog`. I'll update the DB engine specs in the near future.